tsconfig-strict.json

```
@@ -319,6 +319,8 @@
     "core/templates/pages/contributor-dashboard-page/services/contribution-opportunities-backend-api.service.spec.ts",
     "core/templates/pages/contributor-dashboard-page/services/translate-text-backend-api.service.ts",
     "core/templates/pages/contributor-dashboard-page/services/translate-text-backend-api.service.spec.ts",
+    "core/templates/pages/contributor-dashboard-page/services/translate-text.service.ts",
+    "core/templates/pages/contributor-dashboard-page/services/translate-text.service.spec.ts",
```

can you please add them in alphabetical order?
```suggestion
    "core/templates/pages/contributor-dashboard-page/services/translate-text.service.ts",
    "core/templates/pages/contributor-dashboard-page/services/translate-text.service.spec.ts",
    "core/templates/pages/contributor-dashboard-page/services/translate-text-backend-api.service.ts",
    "core/templates/pages/contributor-dashboard-page/services/translate-text-backend-api.service.spec.ts",
```

---

Resolving as per offline discussion